components/merkledb/src/indexes/proof_list/proof.rs

```diff
@@ -14,9 +14,10 @@
 
 pub use crate::ValidationError; // TODO Change for a type alias after EJB switching to rust > 1.36
 
+use anyhow as failure; // FIXME: remove once `ProtobufConvert` derive is improved (ECR-4314)
```

Comment:
Is it really correct issue number ? Or you've meant to improve in a scope of this PR ?

Reply:
I've meant for the Jira issue to correspond to this PR and another PR dealing with this problem, but I can create a new issue just for this FIXME.